### ISSUE TITLE
inspect command doesn't include ipv4/ipv6 IP Addressing

### ISSUE THREAD

**Issue by thinghacker** [srl-labs/containerlab#432]
With containerlab version 0.14.2 when I deploy a lab I can see the IPv4 and IPv6 address information but it is no longer shown when I use the inspect command.

Example is below.

```
adam@m4600-cl:~/simpledc$ sudo containerlab deploy --topo simpledc.yml
INFO[0000] Parsing & checking topology file: simpledc.yml
INFO[0000] Creating lab directory: /home/adam/simpledc/clab-simpledc
INFO[0000] Creating docker network: Name='clab', IPv4Subnet='172.20.20.0/24', IPv6Subnet='2001:172:20:20::/64', MTU='1500'
INFO[0000] Creating container: gateway
INFO[0000] Creating container: leaf1
INFO[0000] Creating container: leaf2
INFO[0000] Creating container: spine1
INFO[0000] Creating container: spine2
INFO[0001] Creating virtual wire: spine1:e1-3 <--> gateway:e1-1
INFO[0001] Creating virtual wire: leaf2:e1-2 <--> spine2:e1-2
INFO[0001] Creating virtual wire: leaf1:e1-1 <--> spine1:e1-1
INFO[0001] Creating virtual wire: spine2:e1-3 <--> gateway:e1-2
INFO[0001] Creating virtual wire: leaf2:e1-1 <--> spine1:e1-2
INFO[0001] Creating virtual wire: leaf1:e1-2 <--> spine2:e1-1
INFO[0002] Writing /etc/hosts file
+---+-----------------------+--------------+--------------------------+---------+-------+---------+----------------+----------------------+
| # |         Name          | Container ID |          Image           |  Kind   | Group |  State  |  IPv4 Address  |     IPv6 Address     |
+---+-----------------------+--------------+--------------------------+---------+-------+---------+----------------+----------------------+
| 1 | clab-simpledc-gateway | f180cd08991d | vrnetlab/vr-sros:21.2.R2 | vr-sros |       | running | 172.20.20.6/24 | 2001:172:20:20::6/64 |
| 2 | clab-simpledc-leaf1   | 7f8008a8c6a3 | srlinux:21.3.1-410       | srl     |       | running | 172.20.20.2/24 | 2001:172:20:20::2/64 |
| 3 | clab-simpledc-leaf2   | 20f587455b78 | srlinux:21.3.1-410       | srl     |       | running | 172.20.20.5/24 | 2001:172:20:20::5/64 |
| 4 | clab-simpledc-spine1  | 609d8fd4f448 | srlinux:21.3.1-410       | srl     |       | running | 172.20.20.3/24 | 2001:172:20:20::3/64 |
| 5 | clab-simpledc-spine2  | 705162c4b5b3 | srlinux:21.3.1-410       | srl     |       | running | 172.20.20.4/24 | 2001:172:20:20::4/64 |
+---+-----------------------+--------------+--------------------------+---------+-------+---------+----------------+----------------------+
```

I can see the IP addresses during initial deployment, but when I run the inspect command, these fields are blank in the table and not provided when using "--format json" either

```
adam@m4600-cl:~/simpledc$ sudo containerlab inspect --all
+---+--------------+----------+-----------------------+--------------+--------------------------+---------+-------+---------+--------------+--------------+
| # |  Topo Path   | Lab Name |         Name          | Container ID |          Image           |  Kind   | Group |  State  | IPv4 Address | IPv6 Address |
+---+--------------+----------+-----------------------+--------------+--------------------------+---------+-------+---------+--------------+--------------+
| 1 | simpledc.yml | simpledc | clab-simpledc-gateway | f180cd08991d | vrnetlab/vr-sros:21.2.R2 | vr-sros |       | running |              |              |
| 2 |              |          | clab-simpledc-leaf1   | 7f8008a8c6a3 | srlinux:21.3.1-410       | srl     |       | running |              |              |
| 3 |              |          | clab-simpledc-leaf2   | 20f587455b78 | srlinux:21.3.1-410       | srl     |       | running |              |              |
| 4 |              |          | clab-simpledc-spine1  | 609d8fd4f448 | srlinux:21.3.1-410       | srl     |       | running |              |              |
| 5 |              |          | clab-simpledc-spine2  | 705162c4b5b3 | srlinux:21.3.1-410       | srl     |       | running |              |              |
+---+--------------+----------+-----------------------+--------------+--------------------------+---------+-------+---------+--------------+--------------+
```

This previously did work in 0.13 but is not functioning in 0.14.1 and 0.14.2

**Comment by karimra** [comment]
Hi, 
Thanks for reporting this.
It should work if you provide the topo file `clab inspect --topo simpledc.yml` 
I will look into why `-a` fails to get the IP addresses

**Comment by thinghacker** [comment]
Hi,
Thanks for looking into this, yes that method seems to work, in my particular case I usually run one lab at a time, so have been going for "--all"

**Comment by hellt** [comment]
@thinghacker the fix has been added in #433 and we will soon release 0.14.2 where `--all` will work again

**Comment by hellt** [comment]
@thinghacker `0.14.2` is available with the mentioned fix
